video_creation/background.py

```diff
@@ -3,29 +3,40 @@
 import re
 from pathlib import Path
 from random import randrange
-from typing import Any, Tuple
+from typing import Any, Tuple,Dict
 
-from moviepy.editor import VideoFileClip
+from moviepy.editor import VideoFileClip,AudioFileClip
 from moviepy.video.io.ffmpeg_tools import ffmpeg_extract_subclip
 from pytube import YouTube
 from pytube.cli import on_progress
 from utils import settings
 from utils.console import print_step, print_substep
+import yt_dlp
 
-# Load background videos
-with open("./utils/backgrounds.json") as json_file:
-    background_options = json.load(json_file)
+def load_background_options():
+    background_options = {}
+    # Load background videos
+    with open("./utils/background_videos.json") as json_file:
+        background_options["video"] = json.load(json_file)
 
-# Remove "__comment" from backgrounds
-background_options.pop("__comment", None)
+    # Load background audios
+    with open("./utils/background_audios.json") as json_file:
+        background_options["audio"] = json.load(json_file)
+    
+    # Remove "__comment" from backgrounds
+    background_options["video"].pop("__comment", None)
+    background_options["audio"].pop("__comment", None)
+    
+    # Add position lambda function
+    # (https://zulko.github.io/moviepy/ref/VideoClip/VideoClip.html#moviepy.video.VideoClip.VideoClip.set_position)
+    for name in list(background_options["video"].keys()):
+        pos = background_options["video"][name][3]
 
-# Add position lambda function
-# (https://zulko.github.io/moviepy/ref/VideoClip/VideoClip.html#moviepy.video.VideoClip.VideoClip.set_position)
-for name in list(background_options.keys()):
-    pos = background_options[name][3]
+        if pos != "center":
+            background_options["video"][name][3] = lambda t: ("center", pos + t)
+
```

Comment:
I don't think it is being used anywhere.

Reply:
This was done in the `master` branch. I think just kept the code and insert it into a function because now i'm gathering the `audio` information from json too.

Reply:
yeah Add TODO of my comment